specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/examples/LogAnalyticsThrottledRequests.json

```diff
@@ -0,0 +1,34 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "api-version": "2017-12-01",
+    "parameters": {
+      "location": "westus",
+      "properties": {
+        "ThrottledRequestsInput": {
+          "blobContainerSasUri": "https://somesasuri",
+          "fromTime": "2018-01-21T01:54:06.862601Z",
+          "toTime": "2018-01-23T01:54:06.862601Z",
+          "groupByOperationName": true,
+          "groupByResourceName": false
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
```

Comment:
In the Swagger, 200 doesn't return a response body.

Reply:
Updated the Swagger file now. 200 should return a response body. Missed the copy from our internal repo.